src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -279,19 +226,21 @@ public void resetEncoder(boolean reset) {
     }
 
     @Config(name = "PID")
-    private void setPID(@Config(name = "P", defaultValueNumeric = P) double p,
+    private void setFixedArmPID(@Config(name = "P", defaultValueNumeric = P) double p,
             @Config(name = "I", defaultValueNumeric = I) double i,
             @Config(name = "D", defaultValueNumeric = D) double d) {
         climbFixedMotor.config_kP(PID_SLOT_0, p);
```

Comment:
This should also check if `climbFixedMotor` is `null`, in the event that we don't have the fixed arm but the subsystem is still instantiated for whatever reason.

Reply:
If we want to guard against `climbFixedMotor` being `null`, then the following places should also have checks added:

- Configuration of `climbFixedMotor` in constructor
- `simulationPeriodic` (#97 added simulation, by the way)
- `encoderPosition`
- `stopFixedPID` (May also want to rename that one)
- `resetEncoder`

Reply:
I was mainly commenting on `setFixedArmPID` because I think Oblog will call it automatically, but it would be good to have `null` checks for the others as well.

The larger issue is that subsystem instantiation isn't tied to hardware nullability - just because `ClimbSubsystem` exists doesn't mean that motors like `climbFixedMotor` aren't `null`, so we essentially have to choose which hardware to check.